apps/json-viewer/src/locations/ConfigScreen.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useCallback, useState, useEffect } from 'react';
+import { AppExtensionSDK } from '@contentful/app-sdk';
+import {
+  Box,
+  Flex,
+  Form,
+  FormControl,
+  Heading,
+  Paragraph,
+  Select,
+} from '@contentful/f36-components';
+import { css } from 'emotion';
+import { /* useCMA, */ useSDK } from '@contentful/react-apps-toolkit';
+
+export interface AppInstallationParameters {}
```

Comment:
I think it'd be good to document what's living in the Installation parameters here

Looks like it's just configOptions with that reactJsonConfig as the value